lib/avrora/resolver.ex

```diff
@@ -100,7 +100,12 @@ defmodule Avrora.Resolver do
             memory_storage().put(schema.id, schema)
           end
 
-        {:error, :unconfigured_registry_url} ->
+        {:error, reason}
+        when reason in [
+               :unconfigured_registry_url,
+               :malformed_registry_auth,
```

Comment:
I don't think that authentication should be handled as a normal case.
Before the change, only when the registry URL was not provided, then it will simply not go to the registry (which is logical), but after the change, the exceptional case will be swallowed. Which can't be considered normal behavior.